lib/interfaceNative.js

```diff
@@ -83,7 +83,7 @@ NativeInterface.prototype.geopos = function(member, zSetName, callBack) {
         if (err) {
             callBack(err, null);
         } else {
-            if (results.lenght > 0) {
+            if (results.length > 0) {
```

Comment:
I've found better solution =)

2016-07-13 10:24 GMT+03:00 Artem Boltunov <temik.fpm@gmail.com>:

> :)))
>
> > +.vs
> > +bin
> > +obj
> > +node-georedis.njsproj
> > +node-georedis.sln
>
> this is from ms visual studio IDE, if you're sure I remove them.
>
>
> 2016-07-12 16:27 GMT+03:00 Arjun Mehta <notifications@github.com>:
>
>> In lib/interfaceNative.js
>> <https://github.com/arjunmehta/node-georedis/pull/13#discussion_r70437095>
>> :
>>
>> > @@ -83,7 +83,7 @@ NativeInterface.prototype.geopos = function(member, zSetName, callBack) {
>> >          if (err) {
>> >              callBack(err, null);
>> >          } else {
>> > -            if (results.lenght > 0) {
>> > +            if (results.length > 0) {
>>
>> Looks good!
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/arjunmehta/node-georedis/pull/13/files/515bce4e4a2dfeee52b0efa68854f34886765a8a..ef000bb98d80f43f922dc7524253eef52c9d1122#r70437095>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/ABrNBj5AxG3EIlEUjXZwbnN_9wLk9muCks5qU5ZPgaJpZM4JJSKn>
>> .
>>
>
>